docs/backend.md

```diff
@@ -4,9 +4,30 @@ This site is built using Drupal and combination of modules and themes.
 ## Drupal
 As of the writing of this document the foundation of the site is built using Drupal + Symphony + Composer. Drupal documentation can be found here: https://www.drupal.org/docs. This site follows Drupal coding standards and best practices.
 
+### Updating Drupal via Composer
+Drupal projects managed through Composer typically use `drupal/core-recommended`, which ensures that Drupal core and its dependencies are installed with secure and tested versions.
+
+For ongoing updates we use the following command `composer update drupal/core-* -W` which will take care of both updating Drupal core any any dependencies that it may require. This will result in an updated `composer.lock` file which will be committed to the repository once the update has been tested locally.
+
+Once updated, the following drush command should be issued, `drush updatedb && drush cr`. This will check for any pending database updates followed up by a clearing of the cache.
```

Comment:
Can you also include a `drush cex` in case configuration was changed as part of the db updates?